src/jvm/main/org/jetbrains/kotlinx/lincheck/runner/InvocationResult.kt

```diff
@@ -27,20 +27,22 @@ class CompletedInvocationResult(
 /**
  * Indicates that the invocation has run into deadlock or livelock found by [ManagedStrategy].
  */
-data object ManagedDeadlockInvocationResult : InvocationResult()
+class ManagedDeadlockInvocationResult(val results: ExecutionResult) : InvocationResult()
```

Comment:
Now that all invocation results contain `results` field, let's propagate this field to the parent class `InvocationResult`.

Reply:
So let's change it to also contain results?)

Then it would be uniform, no special cases, no needs to do type-casts and type-checks (i.e. `result is CompletedInvocationResult`) when we want to access `results` field, etc.